### PR TITLE
Ar 726 recommendations

### PR DESCRIPTION
**Pull request by wissog**
alex said its done
define in env:
REACT_APP_RECOMMENDED=https://kabbalahmedia.info/feed_api/recommend

### CODE REVIEW

File: src/components/Pages/PlaylistCollection/widgets/Playlist/Playlist.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ class PlaylistWidget extends Component {
     }
 
     return (
-      <Header inverted as="h1">
+      <Header as="h1">
```
Comment (gshilin) [inline src/components/Pages/PlaylistCollection/widgets/Playlist/Playlist.js:91]: Это зачем?

File: src/components/Pages/PlaylistCollection/widgets/PlaylistAVBox/PlaylistAVBox.js [hunk 2]
```diff
@@ -172,24 +172,25 @@ class PlaylistAVBox extends Component {
...
+          selected={selected}
+          language={playlist.language}
+          uiLanguage={uiLanguage}
+          onSelectedChange={this.handleSelectedChange}
+          onLanguageChange={this.handleLanguageChange}
+          onSwitchAV={this.handleSwitchAV}
+          history={history}
+        />
+      </Grid.Column>
+      { !embed ?
```
Comment (gshilin) [inline src/components/Pages/PlaylistCollection/widgets/PlaylistAVBox/PlaylistAVBox.js:193]: !embed &&

File: src/components/Pages/Unit/Page.js [hunk 3]
```diff
@@ -98,9 +101,9 @@ export class UnitPage extends Component {
                     <Grid.Column mobile={16} tablet={8} computer={16}>
                       {this.renderDownloads()}
                     </Grid.Column>
-                    <Grid.Column mobile={16} tablet={8} computer={16}>
+                    {/* <Grid.Column mobile={16} tablet={8} computer={16}>
                       {this.renderRecommendations()}
-                    </Grid.Column>
+                    </Grid.Column> */}
```
Comment (gshilin) [inline src/components/Pages/Unit/Page.js:106]: может, удалить?

**Review by gshilin — CHANGES_REQUESTED** [review]

File: src/components/Pages/PlaylistCollection/widgets/Playlist/Playlist.js (hunk 1 above)
Comment (wissog) [reply src/components/Pages/PlaylistCollection/widgets/Playlist/Playlist.js:91]: background there changed, so no need to invert anymore - you don't see the text

File: src/components/Pages/PlaylistCollection/widgets/PlaylistAVBox/PlaylistAVBox.js (hunk 2 above)
Comment (wissog) [reply src/components/Pages/PlaylistCollection/widgets/PlaylistAVBox/PlaylistAVBox.js:193]: possible

File: src/components/Pages/Unit/Page.js (hunk 3 above)
Comment (wissog) [reply src/components/Pages/Unit/Page.js:106]: possible as well, i was not sure whether we need it in the future or not

**Pull request merged**